docs/integrations/destinations/exasol.md

```diff
@@ -0,0 +1,72 @@
+# Exasol
+
+Exasol is the in-memory database built for analytics.
+
+## Sync overview
+
+### Output schema
+
+Exasol tables become Airbyte Streams and Exasol columns become Airbyte Fields. Each table will contain 3 columns:
+
+* `_AIRBYTE_AB_ID`: a uuid assigned by Airbyte to each event that is processed. The column type in Exasol is `VARCHAR(64)`.
+* `_AIRBYTE_DATA`: a json blob representing with the event data. The column type in Exasol is `VARCHAR(2000000)`.
+* `_AIRBYTE_EMITTED_AT`: a timestamp representing when the event was pulled from the data source. The column type in Exasol is `TIMESTAMP`.
+
+
+### Data type mapping
+
+This section should contain a table mapping each of the connector's data types to Airbyte types. At the moment, Airbyte uses the same types used by [JSONSchema](https://json-schema.org/understanding-json-schema/reference/index.html). `string`, `date-time`, `object`, `array`, `boolean`, `integer`, and `number` are the most commonly used data types.
+
+| Integration Type | Airbyte Type | Notes |
+| :--- | :--- | :--- |
```

Comment:
Seems to me missing, still. Is it planned to fill the table during development?

Reply:
Other SQL destinations also don't use this table, so I removed it.